### DISCUSSION TITLE
jdbc链接kyuubi timeout的问题

### DISCUSSION THREAD

**Discussion by stgztsw** [apache/kyuubi#3541] [Q&A]
kyuubi的版本是1.5.1。最近调度系统调度离线任务时，频繁的发生jdbc链接kyuubi超时。这边的所有的离线任务用的是相同jdbc用户调度的。share level是默认的这边没有修改过。当调度任务并发>5的情况下就会出现一些任务链接超时。我看yarn这边只创建了一个sparkdriver来来处理任务，sparkdriver的内存配置为15G。请问链接超时的瓶颈在哪里？是再kyuubi这边？还是在sparkdriver这边？该如何调整

**Comment by pan3793** [comment]
I can not answer your question based on your rough description. There is a known issue https://github.com/apache/incubator-kyuubi/pull/3152 that may be related.
